### DISCUSSION TITLE
Request to increase timeout limit

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1243] [Q&A]
Hello sir, Actually I am satisfied by the services you guys provide. But I face one issue, the timeout limit. As a student I don't have powerful machines to run my builds which cirrus ci provides for free. Since the timeout is 2hr, could you make it only 30 minutes more? I would be very grateful if you increase the timeout limit. It would help me a lot. 
Hoping for a positive reply.

**Comment by fkorotkov** [comment]
Duplicate of #1204
